app/actions/websocket/posts.ts

```diff
@@ -251,6 +253,17 @@ export async function handlePostEdited(serverUrl: string, msg: WebSocketMessage)
     });
     models.push(...postModels);
 
+    // Sync permalink previews that reference this edited post
```

Comment:
Seems like `syncPermalinkPreviewsForEditedPost` explained the comment.  I don't think we'd need the comment.